Police Arrests In Each Neighboorhood:

In [5]:
import pandas as pd

neighborhoods = ["Mount Washington", "East Hills", "Fairywood", "Central Business District", "East Liberty", "Brighton Heights", "Shadyside", "Hazelwood", "South Side Flats", "Squirrel Hill South", "Squirrel Hill North"]

arrestdata = pd.read_csv('arrests.csv')
incident_neighborhood = "INCIDENTNEIGHBORHOOD"
arrestcount = {neighborhood: 0 for neighborhood in neighborhoods}
for index, row in arrestdata.iterrows():
    neighborhood = row[incident_neighborhood]
    if neighborhood in neighborhoods:
        arrestcount[neighborhood] += 1
print("Number of Arrests In Each Neighborhood: ")
print(arrestcount)

Number of Arrests In Each Neighborhood: 
{'Mount Washington': 1434, 'East Hills': 1194, 'Fairywood': 127, 'Central Business District': 4440, 'East Liberty': 1889, 'Brighton Heights': 1114, 'Shadyside': 971, 'Hazelwood': 1396, 'South Side Flats': 3497, 'Squirrel Hill South': 812, 'Squirrel Hill North': 290}


The neighborhood with the least amount of police arrests should be considered the best. Therefore, Fairywood is the best neighborhood as it has 127 police arrests.